file/builder.go

```diff
@@ -771,9 +778,38 @@ func (b *stateBuilder) ingestRoute(r FRoute) error {
 	return b.ingestPlugins(plugins)
 }
 
+func (b *stateBuilder) getPluginSchema(entityID *string) (map[string]interface{}, error) {
+	var schema map[string]interface{}
+
+	// lookup in cache
+	if schema, ok := b.schemas[*entityID]; ok {
+		return schema, nil
+	}
+	schema, err := b.client.Plugins.GetFullSchema(b.ctx, entityID)
```

Comment:
What if client is nil?

Reply:
This is possible with `validate` cmd, no?

Reply:
This method is called by `ingestPluginDefaults`, which does a [check](https://github.com/Kong/deck/blob/DECK-8/file/builder.go#L797-L799) on whether `b.client` is `nil` or not